debug_toolbar/panels/sql/panel.py

```diff
@@ -204,6 +207,30 @@ def process_response(self, request, response):
             if trans_id:
                 self._queries[(i - 1)][1]['ends_trans'] = True
 
+        # Queries are duplicates only if there's as least 2 of them.
+        # Also, to hide queries, we need to give all the duplicate groups an id
+        query_duplicates = {
+            alias: {
+                query: duplicate_count
+                for query, duplicate_count in queries.items()
+                if duplicate_count >= 2
+            }
+            for alias, queries in query_duplicates.items()
+        }
```

Comment:
Until now we haven't used dict comprehensions because we're supporting Django 1.4 and Python 2.5. 1.4 is still supported until October 1st.